wpilib/wpilib/livewindow.py

```diff
@@ -22,48 +33,49 @@ class LiveWindow:
     """The public interface for putting sensors and
     actuators on the LiveWindow."""
 
-    sensors = set()
-    #actuators = set()
     components = {}
-    livewindowTable = None
-    statusTable = None
+    _liveWindowTable = None
+    _statusTable = None
+    _enabledEntry = None
+    startLiveWindow = False
     liveWindowEnabled = False
-    firstTime = True
+    telemetryEnabled = True
+    mutex = threading.RLock()
+
+    @classmethod
+    def liveWindowTable(cls):
+        if cls._liveWindowTable is None:
+            cls._liveWindowTable = NetworkTablesInstance.getDefault().getTable("LiveWindow")
```

Comment:
Just a note: you can use `NetworkTables` instead of `NetworkTablesInstance.getDefault()`.